resources/lib/addon.py

```diff
@@ -553,3 +633,14 @@ def route(self, query):
                 fh.write(traceback.format_exc())
             heading = 'drnu addon crash'
             xbmcgui.Dialog().ok(heading, '\n'.join([tr(30906), tr(30907), tr(30908), crash_file]))
```

Comment:
to fix the merge I think you just need to change 
`xbmcgui.Dialog().ok(heading, '\n'.join([tr(30906), tr(30907), tr(30908), crash_file]))
`
`to xbmcgui.Dialog().ok(heading, '\n'.join([tr(30906), tr(30907), tr(30908), str(crash_file)]))`